pkg/address/getAddress.test.ts

```diff
@@ -0,0 +1,56 @@
+import { getAddress } from "./getAddress"
+import { mocked } from "ts-jest/utils"
+import { Address, getBTCAddress, getBTSAddress } from "./address"
+jest.mock("./address")
+
+describe("getAddress script", () => {
+    beforeEach(() => {
+        jest.resetAllMocks()
+        jest.restoreAllMocks()
```

Comment:
As far as I know you don't need to restore after resetting, but I might be wrong.

Reply:
Indeed, fixed it.